bot/exts/moderation/verification.py

```diff
@@ -519,12 +539,32 @@ async def on_member_join(self, member: discord.Member) -> None:
         if member.guild.id != constants.Guild.id:
             return  # Only listen for PyDis events
 
+        raw_member = await self.bot.http.get_member(member.guild.id, member.id)
+
+        # Only send the message to users going through our gating system
```

Comment:
a little unclear now that we technically have two gating systems.

Reply:
Addressed in ea3217effacc02e06444ea0b21985cd7439a13e7